packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,127 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=tuxguitar
+pkgver=1.5.5
+pkgrel=2
+pkgdesc='Multitrack guitar tablature editor and player'
+arch=(x86_64 i686 armv7h)
+_java_ver=11
+_swt_rel='4.22-202111241800'
+_swt_ver=${_swt_rel:0:-13}
+url='https://sourceforge.net/projects/tuxguitar/'
+license=(LGPL)
+groups=(pro-audio)
+depends=()
+makedepends=(alsa-lib fluidsynth jack "java-environment-openjdk=${_java_ver}" maven)
+optdepends=('alsa-lib: playback with ALSA'
+            'fluidsynth: alternative synthesizer for playback'
+            'jack: playback and MIDI support with JACK'
+            'pulseaudio: playback with PulseAudio')
+conflicts=(tuxguitar-common tuxguitar-gtk2)
+source=("https://downloads.sourceforge.net/tuxguitar/tuxguitar-${pkgver}-src.tar.gz"
+        "https://archive.eclipse.org/eclipse/downloads/drops4/R-${_swt_rel}/swt-${_swt_ver}-gtk-linux-x86_64.zip"
```

Comment:
Do we need to use a different swt distribution for i686 and ARM 32-bit?

Reply:
The last SWT release for i686 was 4.9, which I just pinned in 5b99e0f7f590ae91f8ac176576070c3b8662bfcd